docs/README.md

```diff
@@ -89,78 +89,17 @@ var server = mc.createServer({
   encryption: true,      // optional
   host: '0.0.0.0',       // optional
   port: 25565,           // optional
-  version: '1.16'
+  version: '1.16.3'
 });
+const mcData = require('minecraft-data')(server.version)
+
 server.on('login', function(client) {
-  const w = {
-    piglin_safe: {
-      type: 'byte',
-      value: 0
-    },
-    natural: {
-      type: 'byte',
-      value: 1
-    },
-    ambient_light: {
-      type: 'float',
-      value: 0
-    },
-    infiniburn: {
-      type: 'string',
-      value: 'minecraft:infiniburn_overworld'
-    },
-    respawn_anchor_works: {
-      type: 'byte',
-      value: 0
-    },
-    has_skylight: {
-      type: 'byte',
-      value: 1
-    },
-    bed_works: {
-      type: 'byte',
-      value: 1
-    },
-    has_raids: {
-      type: 'byte',
-      value: 1
-    },
-    name: {
-      type: 'string',
-      value: 'minecraft:overworld'
-    },
-    logical_height: {
-      type: 'int',
-      value: 256
-    },
-    shrunk: {
-      type: 'byte',
-      value: 0
-    },
-    ultrawarm: {
-      type: 'byte',
-      value: 0
-    },
-    has_ceiling: {
-      type: 'byte',
-      value: 0
-    }
-  }
-  client.write('login', {
-    entityId: client.id,
-    levelType: 'default',
-    gameMode: 0,
-    previousGameMode: 255,
-    worldNames: ['minecraft:overworld'],
-    dimensionCodec: {name: '', type:'compound', value: {dimension: {type: 'list', value: {type: 'compound', value: [w]}}}},
-    dimension: 'minecraft:overworld',
-    worldName: 'minecraft:overworld',
-    difficulty: 2,
-    hashedSeed: [0, 0],
-    maxPlayers: server.maxPlayers,
-    reducedDebugInfo: false,
-    enableRespawnScreen: true
-  });
+  
+  let loginPacket = mcData.loginPacket
```

Comment:
Do it the other way round: specify all fields below and extract them all from loginPacket